nginx.conf

```diff
@@ -0,0 +1,18 @@
+worker_processes  1;
+daemon off;
+
+events {
+  worker_connections 1024;
+}
+
+http {
+  server {
+    root /usr/src/app/target;
+    port_in_redirect off;
+    location / {
+      include /etc/nginx/mime.types;
+    }
+    listen 4000;
+    error_page 403 404 /404.html;
+  }
+}
```

Comment:
Comparing with the default nginx.conf; https://github.com/nginxinc/docker-nginx/blob/3ba04e37d8f9ed7709fd30bf4dc6c36554e578ac/mainline/alpine/nginx.conf#L2-L32 customisations are:

```
port_in_redirect off;
listen 4000;
error_page 403 404 /404.html;
```

We could try putting those in a file, and adding to the image at; `/etc/nginx/conf.d/000-default.conf` (and the main configuration file will include that file).